SOS/APP/app.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * app.h
+ *
+ * Created: 6/1/2023 7:26:28 AM
+ *  Author: Alaa
+ */ 
+
+
+#ifndef APP_H_
+#define APP_H_
+
+#define APP_LED_0_PORT				LED_PORT_B

```

Comment:
Move this to a header file that includes the board pins configurations. This is not an App Logic

Reply:
@MinaRaoufAziz I am sorry can you elaborate more on what we can do here, this concept might be new to us, do we make a new board pins config header file `board_pin_config.h` for these? where do we put it? LIB folder?